.github/workflows/pr.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     name: Static Checks
     uses: ./.github/workflows/static-checks.yml
     with:
-      a11y-exit: true
+      a11y-exit: false
```

Comment:
We can do this in a follow-up

Reply:
I agree we should address the issues identified, or ignore if they don't make sense. I changed the value of that property because since the property existed I imagined the reason for its existence was this: allow pipeline to pass while the a11y are not fixed.
Changing the value back would mean the issues will need to be fixed in this PR.

Reply:
Please check new commit.